shell.py

```diff
@@ -178,7 +163,8 @@ def run_task_in_region(task='test', region_name=default_region(), parallel=True,
     # this function doesn't actually work for multiple regions
     # and is unlikely to work in general
     # so it fits perfectly with the code 'round here...
-    ip_list = get_pid_ips(pids)
+    # edit: now it works :)
```

Comment:
Then remove the snarky comment, rather than adding to it. :P

Also, iirc this is what was here _before_ I changed it, I have no idea why, but it didn't work even more. I think there were problems with inconsistent ordering? Anyway, that might have only been relevant for the test I have been running then, if it works for your tests then it's alright. Lets just stop using this code asap...

Reply:
I like this snarky comment and it shows how well this code is evolving :3

Hmmm, but i do not think ordering is important here. This does not do anything depending on pid or ip ordering. Anyway, does not matter much. If it breaks probably someone will fix it next time and add another line to the comment.